content/en/docs/polo-features/callsign-stacking/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Callsign Stacking"
+weight: 132
+description: The PoLo method for handling pile-ups
+---
+Everyone will have their favourite way to handle a pileup. For those who like the callsign fragment method i.e. one where you might scribble down callsign fragments and then back track to each fragment in the list one at a time, then Callsign Stacking may be of interest.
+
+In the "Their Call" entry field, enter each fragment that is heard separated by double slashes or two dots which will be converted to slashes.
+
+![image](./stacking-1.png)
+
+Then work backwards through the stack, correcting each call until confirmed. Once confirmed, send the corrected call to your log.
```

Comment:
we might want to expand the "send the corrected call" and make it clear that it just means "press enter or tap the logging button"